ansible/vars/epoch/uat.yml

```diff
@@ -1,6 +1,11 @@
 api_base_uri: http://{{ public_ipv4 }}:{{ epoch_config.http.external.port }}/v2
-
 configure_peers: false
+seed_peers:
+  - "52.10.46.160"
+  - "18.195.109.60"
+  - "13.250.162.250"
+  - "18.130.148.7"
```

Comment:
I see same 4 as at https://github.com/aeternity/epoch/blob/f7a0d9d38c372c2b37b2d6a19fa6768ee4804e64/docs/release-notes/RELEASE-NOTES-0.25.0.md#inspect-the-testnet
Ok.